### PR TITLE
Edit subitems

### PR DESCRIPTION
**Pull request by pconesa**
- Edit set protocol works with complex sets (titlseries, Classes2d) using "Sub item

### CODE REVIEW

File: pwem/protocols/protocol_set_editor.py [hunk 1]
```diff
@@ -51,31 +51,53 @@ def _defineParams(self, form):
...
 
     def formulaStep(self):
         """
         Goes through all items in the input set and applies the formula to each of them using exec.
         Complex python code could be run separating lines with ;  To use numpy you could do
         import numpy; item._resolution.set(numpy.random.randint(10))
         """
+
+        import time
+        time.sleep(10)
```
Comment (JorMaister) [inline pwem/protocols/protocol_set_editor.py:78]: Why is this needed?

Comment (pconesa) [reply pwem/protocols/protocol_set_editor.py:78]: To see if you review the code!! Was intentional!! 😜

**Review by JorMaister — APPROVED** [review]
